test/ci-operator-configresolver-integration/run.sh

```diff
@@ -14,11 +14,35 @@ trap "rm -rf ${WORKDIR}" EXIT
 cd $WORKDIR
 # copy registry to tmpdir to allow tester to modify registry
 cp -a $ROOTDIR/test/ci-operator-configresolver-integration/ tests
-ci-operator-configresolver -config tests/configs -log-level debug &
+ci-operator-configresolver -config tests/configs -log-level debug -cycle 2m &
 PID=$!
+# generation should be >= 1. 0 or an error means the config function has not run yet
+if ! timeout 10s bash -c 2>/dev/null -- 'until [[ $(curl http://127.0.0.1:8080/generation 2>/dev/null) -ne 0 ]]; do sleep 1; done'; then
+    echo "Timed out waiting for generation to equal 1"
+    kill $PID
+    wait $PID
+    exit 1
+fi
 if ! timeout 10s bash -c 2>/dev/null -- "until diff tests/expected/openshift-installer-release-4.2.json <(curl 'http://127.0.0.1:8080/config?org=openshift&repo=installer&branch=release-4.2'); do sleep 1; done" &>/dev/null; then
     echo "diff: $(diff -ru tests/expected/openshift-installer-release-4.2.json <(curl 'http://127.0.0.1:8080/config?org=openshift&repo=installer&branch=release-4.2'))"
     kill $PID
+    wait $PID
+    exit 1
+fi
+export currGen=$(curl 'http://127.0.0.1:8080/generation')
+cp tests/configs2/release-4.2/openshift-installer-release-4.2-golang111.yaml tests/configs/release-4.2/openshift-installer-release-4.2.yaml
+if ! timeout 10s bash -c 2>/dev/null -- 'until [[ $(curl http://127.0.0.1:8080/generation 2>/dev/null) -eq $currGen+1 ]]; do sleep 1; done'; then
+    echo "Generation is $(curl http://127.0.0.1:8080/generation)"
+    echo "Timed out waiting for generation to equal $nextGen"
```

Comment:
What is `$nextGen`?

Reply:
An old variable that I meant to change here